drizzle-orm/src/sqlite-core/dialect.ts

```diff
@@ -660,8 +660,8 @@ export class SQLiteSyncDialect extends SQLiteDialect {
 		const dbMigrations = session.values<[number, string, string]>(
 			sql`SELECT id, hash, created_at FROM "__drizzle_migrations" ORDER BY created_at DESC LIMIT 1`,
 		);
-
-		const lastDbMigration = dbMigrations[0] ?? undefined;
+		// fix for bun-sqlite migrations where dbMigrations return null
+		const lastDbMigration = dbMigrations == null ? undefined : dbMigrations[0] ?? undefined;
```

Comment:
```suggestion
		const lastDbMigration = dbMigrations?.[0] ?? undefined;
```

Reply:
Thanks for the edit, it's way cleaner that way